src/components/Skeletons/SearchRowSkeleton.tsx

```diff
@@ -96,56 +97,86 @@ function TableListItemSkeleton({shouldAnimate = true, fixedNumItems}: TableListI
             shouldAnimate={shouldAnimate}
             fixedNumItems={fixedNumItems}
+            gradientOpacityEnabled={gradientOpacityEnabled}
+            itemViewStyle={[styles.highlightBG, styles.mb3, styles.br3, styles.mr3, styles.ml3]}
             renderSkeletonItem={() => (
                 <>
                     <Rect
-                        x="20"
-                        y="10"
-                        rx="5"
-                        ry="5"
-                        width="40"
-                        height="40"
+                        x={12}
+                        y={12}
+                        rx={5}
+                        ry={5}
+                        width={36}
+                        height={40}
                     />
                     <Rect
-                        x="80"
-                        y="25"
-                        width={shortBarWidth}
-                        height={barHeight}
-                    />
-                    <Rect
-                        x="150"
-                        y="25"
-                        width={longBarWidth}
+                        x={60}
+                        y={28}
+                        width={30}
                         height={barHeight}
                     />
                     <Rect
-                        x="320"
-                        y="25"
+                        x={102}
+                        y={28}
                         width={longBarWidth}
                         height={barHeight}
                     />
+                    {isLargeScreenWidth && (
+                        <>
+                            <Rect
+                                x={234}
+                                y={28}
+                                width={longBarWidth}
+                                height={barHeight}
+                            />
+
+                            <Rect
+                                x={366}
+                                y={28}
+                                width={60}
+                                height={barHeight}
+                            />
+                        </>
+                    )}
+
                     <Rect
-                        x="480"
-                        y="25"
-                        width={longBarWidth}
+                        // We have to calculate this value to make sure the element is aligned to the button on the right side. Below is the calculation:
+                        // 375 is the width of the left pane
+                        // 80 is the width of the button on the right side
+                        // 12 is the margin between the element and the right border
+                        // 24 is the padding of the central pane summing two sides
+                        // 12 is the gap between the element and the right button
+                        // 80 is the width of the element itself
+                        x={windowWidth - 375 - 80 - 12 - 24 - 12 - 80}
```

Comment:
Let's define all of those numbers as constants since we're using them more than once

Reply:
Done!